docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md

```diff
@@ -0,0 +1,74 @@
+
+## Prerequisites
+Your subaccount must have [entitlements](https://help.sap.com/docs/btp/sap-business-technology-platform/managing-entitlements-and-quotas-using-cockpit) for [SAP BTP, Kyma runtime](https://discovery-center.cloud.sap/index.html#/serviceCatalog/kyma-runtime) and [SAP Cloud Management Service for SAP BTP
+](https://discovery-center.cloud.sap/index.html#/serviceCatalog/cloud-management-service).
+
+### CLI tools
+- [jq](https://jqlang.github.io/jq/)
+- [curl](https://curl.se/)
+- [btp CLI](https://help.sap.com/docs/btp/sap-business-technology-platform/download-and-start-using-btp-cli-client?locale=en-US) (optional)
+
+## Steps
+
+1. Provision SAP Cloud Management Service instance with `local` plan and create a binding to get the credentials for Provisioning Service API. The procedure is described in [Help Portal](https://help.sap.com/docs/btp/sap-business-technology-platform/getting-access-token-for-sap-cloud-management-service-apis) or you can use btp CLI as shown below:
+
+   1. Set `CIS_INSTANCE_NAME` environment variable with the name of the SAP Cloud Management Service instance:
+      ```bash
+      export CIS_INSTANCE_NAME={CIS_INSTANCE_NAME}
+      ```
+   2. Provision SAP Cloud Management Service instance with Client Credentials grant type passed as parameters: 
+      ```bash
+      btp create services/instance --offering-name cis --plan-name local --name ${CIS_INSTANCE_NAME} --parameters {\"grantType\":\"clientCredentials\"}
+      ```
+   3. Create a binding for the instance:
+      ```bash
+      btp create services/binding --name ${CIS_INSTANCE_NAME}-binding --instance-name ${CIS_INSTANCE_NAME}
+      ```
+
+2. Set `CLIENT_ID`, `CLIENT_SECRET`, `UAA_URL`, `PROVISIONING_SERVICE_URL` environment variables using the credentials from the binding stored in `clientid`, `clientsecret`, `url`, `provisioning_service_url` fields. You can use btp CLI to get the credentials as shown below:
+   ```bash
+   export CLIENT_ID=$(btp --format json get services/binding --name cis-local-binding | jq -r '.credentials.uaa.clientid')
+   export CLIENT_SECRET=$(btp --format json get services/binding --name cis-local-binding | jq -r '.credentials.uaa.clientsecret')
+   export UAA_URL=$(btp --format json get services/binding --name cis-local-binding | jq -r '.credentials.uaa.url')
+   export PROVISIONING_SERVICE_URL=$(btp --format json get services/binding --name cis-local-binding | jq -r '.credentials.endpoints.provisioning_service_url')
+   ```
+
+3. Get the access token for Provisioning Service API using the client credentials:
+   ```bash
+   TOKEN=$(curl -s -X POST "${UAA_URL}/oauth/token" -H "Content-Type: application/x-www-form-urlencoded" -u "${CLIENT_ID}:${CLIENT_SECRET}" --data-urlencode "grant_type=client_credentials" | jq -r '.access_token')
+   ```
+
+4. Check if Kyma runtime is available for provisioning:
```

Comment:
provide payload in this point, all pf the parameters are const

Reply:
I added environment variables with const values in the bash command.